docker-files/Dockerfile

```diff
@@ -20,10 +20,14 @@ FROM maven:3.3-jdk-8-alpine
 
 COPY docker-files/pom.xml .
 
-RUN mvn dependency:copy -q
+RUN mvn dependency:copy-dependencies -q
```

Comment:
I don't know about the change itself, but I assume it need to be done in Dockerfile.aarch64 also.